lib/Mongo/MongoCollection.php

```diff
@@ -96,6 +110,19 @@ public function __get($name)
         return $this->db->selectCollection($this->name . '.' . $name);
     }
 
+    public function __set($name, $value)
+    {
+        if ('w' == $name) {
```

Comment:
I know this is a case of "but I like this better", but I'd prefer not using Yoda conditions. Thanks!